src/arrayMethodSort.js

```diff
@@ -5,7 +5,26 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    compareFunction = compareFunction || function(a,b){
```

Comment:
Consider providing a more descriptive name for the default compare function. The current name `compareFunction` is fine when it's passed by the user, but for the internal default function, a name like `defaultCompare` might be clearer.